### ISSUE TITLE
Bug in SQL query in QueryWithSubdocuments sample?

### ISSUE THREAD

**Issue by MichelBarnett** [Azure/azure-documentdb-dotnet#160]
I'm running the QueryWithSubdocuments method found here:

https://github.com/Azure/azure-documentdb-dotnet/blob/master/samples/code-samples/Queries/Program.cs 

If I run the query in that sample using Query Explorer in the Azure portal I get this result:

Query: SELECT c FROM c in f.Children
Result:

[
  {
    "c": {
      "FamilyName": null,
      "FirstName": "Henriette Thaulow",
      "Gender": "female",
      "Grade": 5,
      "Pets": [
        {
          "GivenName": "Fluffy"
        }
      ]
    }
  },
  {
    "c": {
      "FamilyName": "Merriam",
      "FirstName": "Jesse",
      "Gender": "female",
      "Grade": 8,
      "Pets": [
        {
          "GivenName": "Goofy"
        },
        {
          "GivenName": "Shadow"
        }
      ]
    }
  },
  {
    "c": {
      "FamilyName": "Miller",
      "FirstName": "Lisa",
      "Gender": "female",
      "Grade": 1,
      "Pets": null
    }
  }
]

However, this code in the sample:

```
        var childrenSqlQuery = client.CreateDocumentQuery<Child>(collectionLink,
            "SELECT c " +
            "FROM c IN f.Children").ToList();

        foreach (var child in childrenSqlQuery)
        {
            Console.WriteLine(JsonConvert.SerializeObject(child));
        }
```

Generates this result:

{"FamilyName":null,"FirstName":null,"Gender":null,"Grade":0,"Pets":null}
{"FamilyName":null,"FirstName":null,"Gender":null,"Grade":0,"Pets":null}
{"FamilyName":null,"FirstName":null,"Gender":null,"Grade":0,"Pets":null}

In other words the code correctly returns three objects but all properties are NULL.  

Note that the Linq query also found in QueryWithsubdocuments() does return the correct result.

Is this an SDK bug?

**Comment by arramac** [comment]
This is a bug in the samples (not SDK). The SQL query should've been SELECT VALUE c, which de-serializes to Child correctly. Fixed with https://github.com/Azure/azure-documentdb-dotnet/pull/162
